### PR TITLE
Fix/code duplication for services calls

### PR DESCRIPTION
**Pull request by mathieurobert1**
I delete duplication codes for services and fix some responsive pages.

### CODE REVIEW

**Review by theophile-jr — DISMISSED** [review]

File: angular-frontend/src/app/pages/astrological-compatibility/astrological-compatibility.component.ts [hunk 1]
```diff
@@ -60,16 +49,20 @@ export class AstrologicalCompatibilityComponent {
...
-            },
-            (error) => { console.error("Failed to load compatility", error); }
-        );
+        // this.compatibilityService.getCompatibility(this.customerLeft.id, this.customerRight.id).subscribe(
+        //     (data) => {
+        //         this.compatibility = data;
+        //         this.compatibilityPercentage = this.compatibility.compatibilityScore;
+        //     },
+        //     (error) => { console.error("Failed to load compatility", error); }
+        // );
```
Comment (Njord201) [inline angular-frontend/src/app/pages/astrological-compatibility/astrological-compatibility.component.ts:61]: Remove this comment.

File: angular-frontend/src/app/pages/client-profile/client-profile.component.ts [hunk 2]
```diff
@@ -31,33 +31,28 @@ export class ClientProfileComponent {
...
-            (error) => { console.error("Failed to load encounters list", error); }
-        );
+        // this.paymentHistoryService.getPaymentsCustomer(this.customer.id).subscribe(
+        //     (data) => { this.payments = data; console.log(data)},
+        //     (error) => { console.error("Failed to load payments list", error); }
+        // );
+        // this.encountersService.getCustomerEncounters(this.customer.id).subscribe(
+        //     (data) => { this.encounters = data; console.log(`Encounters : ${data}`)},
+        //     (error) => { console.error("Failed to load encounters list", error); }
+        // );
```
Comment (Njord201) [inline angular-frontend/src/app/pages/client-profile/client-profile.component.ts:53]: Remove this comment.

File: angular-frontend/src/app/pages/coaches/coaches.component.ts [hunk 3]
```diff
@@ -42,10 +33,11 @@ export class CoachesComponent {
...
         for (let key in this.customerToSave) {
             let customer = this.customerToSave[key];
-            this.customersService.updateCustomer(customer.id, customer).subscribe(
-                null,
-                (error) => { console.error("Failed to update customer", error); }
-            );
+            // this.customersService.updateCustomer(customer.id, customer).subscribe(
+            //     null,
+            //     (error) => { console.error("Failed to update customer", error); }
+            // );
```
Comment (Njord201) [inline angular-frontend/src/app/pages/coaches/coaches.component.ts:39]: Remove this comment.

File: angular-frontend/src/app/pages/register-employee/register-employee.component.ts [hunk 4]
```diff
@@ -48,12 +48,16 @@ export class RegisterEmployeeComponent {
...
-                this.router.navigate(["/coaches"]);
-            },
-            (error) => {console.error("Failed to create New Employee", error)}
-        );
+        // this.employeesService.createEmployee(this.newEmployee).subscribe(
+        //     () => {
+        //         this.router.navigate(["/coaches"]);
+        //     },
+        //     (error) => {console.error("Failed to create New Employee", error)}
+        // );
```
Comment (Njord201) [inline angular-frontend/src/app/pages/register-employee/register-employee.component.ts:56]: Remove this comment.

File: angular-frontend/src/app/service/auth/auth.service.ts [hunk 5]
```diff
@@ -60,21 +61,29 @@ export class AuthService {
...
+        //             localStorage.setItem("Manager", "true");
+        //         } else {
+        //             localStorage.setItem("Manager", "false");
+        //         }
+        //     },
+        //     (error) => {
+        //         console.log("Failed to get Me employee", error);
+        //         localStorage.setItem("Manager", "false");
+        //     }
+        // );
```
Comment (Njord201) [inline angular-frontend/src/app/service/auth/auth.service.ts:78]: Remove this comment.

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by theophile-jr — APPROVED** [review]

**Review by Njord201 — APPROVED** [review]
wow big BG

**Pull request merged**